packages/zapp/console/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@flyteconsole/client-app",
-    "version": "1.1.3",
+    "version": "0.0.0-develop",
```

Comment:
@evalsocket do we plan to alwyay have `0.0.0-develop` as a version here? 
In flyteconsole we show which flyteconsole version is used by prebuilding app version from package.json (which means we would show it as 0.0.0-develop in local host.
 Will this package version be changed during release PR prior to build to a proper version?

Reply:
We will replace it with correct version in docker build https://github.com/flyteorg/flyteconsole/pull/529/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R49